apiserver/facades/controller/instancepoller/merge.go

```diff
@@ -69,19 +69,23 @@ func (o *mergeMachineLinkLayerOp) processExistingDevice(dev networkingcommon.Lin
 	// Match the incoming device by hardware address in order to
 	// identify addresses by device name.
 	// Not all providers (such as AWS) have names for NIC devices.
-	incomingDev := o.Incoming().GetByHardwareAddress(dev.MACAddress())
+	incomingDevs := o.Incoming().GetByHardwareAddress(dev.MACAddress())
```

Comment:
Can we somehow make it clear that we operate on a normalized version of the incoming devices? Perhaps by having a `NormalizedIncoming` method name or naming the variable `incomingNormalizedDevs` or some explicit comment here? I had to go back and see where this is set to understand the comment on L85.

Reply:
I attempted to clarify the comment.